cmd/webhook/main.go

```diff
@@ -82,6 +83,23 @@ func main() {
 		namespace = "kube-system"
 	}
 
+	if *healthCheckPort < 1024 || *healthCheckPort > 65535 {
```

Comment:
If we want to don't allow 1-1023 ports usage it should be documented

Reply:
these are regarded as "well known ports" and should not be used (reserved by IANA).

no need to document IMO